adding 3 words to test pulling

LucyFerMon opened the pull request:
Just three words at the end of the vignette

LucyFerMon committed:
Merge remote-tracking branch 'joebrew/master' into develop

LucyFerMon committed:
Testing committing capabilities

joebrew commented:
Looks good, Lucia. Let's perform "code reviews" for all merges. So, I won't merge anything unless I get someone else to "approve" (like you), and vice-versa.

Also, you should request a "reviewer" for any pull request. And the person who issues the pull request should be the one to ultimately merge it.

I've added myself as a reviewer.

Oftentimes, in code reviews, the result is LOTS of changes to the code. That's a good thing.

In this case, however, I approve!

joebrew reviewed (approved):
I approve as is.

LucyFerMon committed:
Merge remote-tracking branch 'joebrew/develop' into feature_ento

LucyFerMon committed:
Merge remote-tracking branch 'joebrew/develop' into develop

LucyFerMon commented:
Love the idea!! Really!

Can the person that issues the pull (me in this case) can ultimate merge it into the develop branch?? Isn't it just you as project owner who can do it? Do I have write access? And if so, how do I see when  have write access to a project? thanks!

LucyFerMon committed:
Committing from feature_ento to my develop

Just learning

LucyFerMon committed:
Adding commits to my develop branch

# Conflicts:
#	vignettes/vignette.Rmd

LucyFerMon committed:
commit from feature_ento local to feature_ento remote

LucyFerMon committed:
commiting changes I did in develop to feature_ento

I had to do this because there was a conflict when I tried to update
feature_ento with develop’s content

LucyFerMon committed:
Commit after trying to sync with develop

LucyFerMon committed:
commit to try sun develop with feature_ento again

LucyFerMon committed:
Merge branch 'feature_ento' into develop

LucyFerMon committed:
Commit to feauture_ento

Learning

LucyFerMon committed:
commit to develop

learning

LucyFerMon committed:
update of feature_ento from develop

learning

LucyFerMon committed:
commit to try pull request

LucyFerMon committed:
Merge pull request #1 from LucyFerMon/feature_ento into develop branch

Pull request on feature_ento to develop
